incubation-openmetrics.md

```diff
@@ -0,0 +1,46 @@
+# OpenMetrics proposal for CNCF incubation
+
+## Background
+
+OpenMetrics specifies the de-facto standard for transmitting cloud-native metrics at scale, with support for both text representation and Protocol Buffers. It’s the wire format and data model Prometheus, Cortex, and Thanos rely on and are migrating more componentes to.
+
+At the same time, OpenMetrics is pursuing an RFC under IETF OPSAWG https://datatracker.ietf.org/doc/draft-richih-opsawg-openmetrics/ . 
+
+OpenMetrics was created outside of Prometheus at the request of Dan Kohn and Chris Aniszczyk in 2015 and named OpenMetrics in 2016.
+
+OpenMetrics came out 5th place in the CNCF End User Survey on Observability tools.
+
+OpenMetrics entered the CNCF Sandbox in July 2018.
+
+## Progress since joining the CNCF Sandbox
+
+OpenMetrics carefully evolved and tightened Prometheus exposition format 0.0.4.
+
+Here are some highlights:
+
+ * Prometheus has supported and used OpenMetrics on the wire since 2018
+ * Cortex and Thanos support it
+ * DataDog has reimplemented OpenMetrics on their own without help from OpenMetrics
+ * OpenMetrics has been submitted to IETF OPSAWG
+ * IETF OPSAWG showed very strong interest in quickly releasing OpenMetrics after a presentation at IETF 110
```

Comment:
nit: For consistency add full stops to the end of each of these to match the last bullet point?

Reply:
Will do.